shared/utils/combine-ovals.py

```diff
@@ -271,6 +271,20 @@ def check_is_loaded(loaded_dict, filename, version):
     return False
 
 
+def check_oval_version_from_oval(xml_content, oval_version):
+    oval_file_tree = ElementTree.fromstring(_header("", "") + xml_content + footer)
+    for defgroup in oval_file_tree.findall("./{%s}def-group" % oval_ns):
```

Comment:
could we fail if there are 2 def-groups of the same oval version?

Reply:
We currently only parse in a single def-group at a time. For multi def-group handling, I say that we should handle this in the future.... or am I miss understanding something again.